adapters/sharethrough/butler.go

```diff
@@ -90,11 +92,13 @@ func (s StrOpenRTBTranslator) requestFromOpenRTB(imp openrtb.Imp, request *openr
 	var height, width uint64
 	if len(strImpParams.IframeSize) >= 2 {
 		height, width = uint64(strImpParams.IframeSize[0]), uint64(strImpParams.IframeSize[1])
-	} else {
+	} else if imp.Banner != nil {
 		height, width = s.Util.getPlacementSize(imp.Banner.Format)
+	} else {
+		height, width = 1, 1
 	}
```

Comment:
Do you think I'd be desirable to add a test to assert the new logic implemented in regards of the `height` and `width` fields?